### PR TITLE
migrate from Travis CI to GitHub Actions workflow

### PR DESCRIPTION
**Pull request by astj**
Now Travis CI's build credit is so small that we cannot run efficient tests for this repo. (ref: https://travis-ci.com/github/mackerelio/cookbook-mackerel-agent/builds/232524924 )

So I'd like to migrate GitHub Actions.

And also, I removed CentOS 6 CI since it reached its EOL and CI did not pass. Though there're other CI conditions that reached their EOLs, but I kept them for now. (hope we will adopt them in future…)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cookbook-mackerel-agent [![Build Status](https://travis-ci.org/mackerelio/cookbook-mackerel-agent.svg?branch=master)](https://travis-ci.org/mackerelio/cookbook-mackerel-agent)
+cookbook-mackerel-agent [Build Status](https://github.com/mackerelio/cookbook-mackerel-agent/workflows/Test/badge.svg)](https://github.com/mackerelio/cookbook-mackerel-agent/actions?workflow=Test)
```
Comment (yseto) [inline README.md:1]: The badge part seemed to be collapsing.

```suggestion
cookbook-mackerel-agent [![Build Status](https://github.com/mackerelio/cookbook-mackerel-agent/workflows/Test/badge.svg)](https://github.com/mackerelio/cookbook-mackerel-agent/actions?workflow=Test)
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      matrix:
+        ruby-version: ['2.3']
+        dist:
+          - ubuntu-14.04
+          - ubuntu-16.04
+          - debian-8
+          - debian-9
+          - centos-7
+          - amazonlinux
+          - amazonlinux-2
```
Comment (yseto) [inline .github/workflows/ci.yml:23]: Is there a reason why ubuntu-18.04 be disappearing?

**Review by yseto — COMMENTED** [review]
seems good.
please check comments.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (astj) [reply .github/workflows/ci.yml:23]: Oh, I just forgot it 😇

**Pull request merged**